### PR TITLE
Add aria text to offical gov't tagline

### PR DESCRIPTION
**Pull request by BrewCityBoy**
Closes Design-Development/Design-and-Content-Team/issues/144

---

added tag: aria-label="An official website of the United States government"

### CODE REVIEW

**Review by anselmbradford — CHANGES_REQUESTED** [review]
So the aria-label should actually be "Official website of the United States government"

The issue was screen-readers were making "An official …" sound like "Unofficial"

File: cfgov/v1/jinja2/v1/includes/atoms/tagline.html [hunk 1]
```diff
@@ -14,7 +14,8 @@
    ========================================================================== #}
 
 {% macro render( is_large ) %}
-<div class="a-tagline{{ ' a-tagline__large' if is_large else '' }}">
+<div class="a-tagline{{ ' a-tagline__large' if is_large else '' }}" 
+      aria-label="{{ _('Official website of the United States government') }}">
```
Comment (anselmbradford) [inline cfgov/v1/jinja2/v1/includes/atoms/tagline.html:18]: Super trivial: Let's align the `class` and `aria-label` … or did your tabbing settings get it to here? Prob best to align it.

Comment (chosak) [reply cfgov/v1/jinja2/v1/includes/atoms/tagline.html]: I just pushed a commit to fix the backend translation piece, which I think fixes this as well.

File: cfgov/v1/locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -42,10 +42,14 @@ msgid "Consumer Financial Protection Bureau"
 msgstr "Oficina para la Protección Financiera del Consumidor"
 
 #: v1/jinja2/v1/includes/atoms/tagline.html:18
+msgid "Official website of the"
+msgstr "Un sitio web oficial del"
+
+#: v1/jinja2/v1/includes/atoms/tagline.html:21
 msgid "An official website of the"
 msgstr "Un sitio web oficial del"
```
Comment (anselmbradford) [inline cfgov/v1/locale/es/LC_MESSAGES/django.po:50]: We don't want this twice @chosak ?

**Review by anselmbradford — APPROVED** [review]
LGTM!

**Pull request merged**